src/sagemaker/workflow/README.rst

```diff
@@ -0,0 +1,162 @@
+You can get it here: https://github.com/villasv/aws-airflow-stack
+
+Using Airflow SageMaker Operators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting from Airflow 1.10.1, you could use SageMaker operators in Airflow. All SageMaker operators take a
+configuration dictionary that can be easily generated by SageMaker Python SDK. For example:
+
+.. code:: python
+
+    import sagemaker
+    from sagemaker.tensorflow import TensorFlow
+    from sagemaker.workflow.airflow import training_config, transform_config_from_estimator
+
+    estimator = TensorFlow(entry_point='tf_train.py',
+                           role='sagemaker-role',
+                           framework_version='1.11.0',
+                           training_steps=1000,
+                           evaluation_steps=100,
+                           train_instance_count=2,
+                           train_instance_type='ml.p2.xlarge')
+
+    # train_config specifies SageMaker training configuration
+    train_config = training_config(estimator=estimator,
+                                   inputs=your_training_data_s3_uri)
+
+    # trans_config specifies SageMaker batch transform configuration
+    trans_config = transform_config_from_estimator(estimator=estimator,
+                                                   instance_count=1,
+                                                   instance_type='ml.m4.xlarge',
+                                                   data=your_transform_data_s3_uri,
+                                                   content_type='text/csv')
+
+Now we can pass these configurations to related SageMaker operators and create the workflow:
+
+.. code:: python
+
+    import airflow
+    from airflow import DAG
+    from airflow.contrib.operators.sagemaker_training_operator import SageMakerTrainingOperator
+    from airflow.contrib.operators.sagemaker_transform_operator import SageMakerTransformOperator
+
+    default_args = {
+        'owner': 'airflow',
+        'start_date': airflow.utils.dates.days_ago(2),
+        'provide_context': True
+    }
+
+    dag = DAG('tensorflow_example', default_args=default_args,
+              schedule_interval='@once')
+
+    train_op = SageMakerTrainingOperator(
+        task_id='training',
+        config=train_config,
+        wait_for_completion=True,
+        dag=dag)
+
+    transform_op = SageMakerTransformOperator(
+        task_id='transform',
+        config=trans_config,
+        wait_for_completion=True,
+        dag=dag)
+
+    transform_op.set_upstream(train_op)
+
+Using Airflow Python Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Airflow PythonOperator <https://airflow.apache.org/howto/operator.html?#pythonoperator>`_
+is a built-in operator that can execute any Python callables. If you want to build the SageMaker workflow in a more
+flexible way,  you could write your python callables for SageMaker operations using SageMaker Python SDK. For example:
```

Comment:
...flexible way, writer your python callables for SageMaker operatoins by using the SageMaker Python SDK.

Reply:
Updated.